sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md

```diff
@@ -2987,6 +3007,219 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -DocumentUnderstandingModelScope
+
+This parameter allows administrators to limit which SharePoint sites the document understanding model and [unstructurted document processesing](https://docs.microsoft.com/en-US/microsoft-365/syntex/document-understanding-overview?WT.mc_id=365AdminCSH_inproduct) premium feature is available on.
```

Comment:
>https://docs.microsoft.com/en-US [](http://example.com/codeflow?start=144&length=32)

this applies to all your URLs:

1. use relative file paths. e.g. this should be `/microsoft-365/syntex/document-understanding-overview`

2. remove the tracking query param (?WT.mc_id...) -- this is dynamically added in the UX product to track when users click these links from within the UX.